devicetypes/smartthings/qubino-flush-1d-relay.src/qubino-flush-1d-relay.groovy

```diff
@@ -0,0 +1,402 @@
+	commands << zwave.associationV1.associationSet(groupingIdentifier:4, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:5, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:6, nodeId:[zwaveHubNodeId]).format()
+	delayBetween(commands, 500)
+}
+
+
+def initialize() {
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def getCommandClassVersions() {
+	[
+		0x20: 1, //Basic
+		0x5A: 1, //Device Reset Locally
+		0x73: 1, //Powerlevel
+		0x86: 2, //Version
+		0x72: 2, //Manufacturer Specific
+		0x27: 1, //Switch All
+		0x25: 1, //Switch Binary
+		0x32: 4, //Meter
+		0x31: 5, //Sensor Multi Level 
+		0x85: 2, //Association
+		0x8E: 3, //Multi Channel Association
+		0x59: 2, //Association Grp Info
+		0x70: 2, //Configuration
+	]
+}
+
+def parse(String description) {
+	log.debug "parse() description: ${description}"
+	def result = null
+	def cmd = zwave.parse(description, commandClassVersions)
+	if (cmd) {
+		result = zwaveEvent(cmd)
+		log.debug("'$description' parsed to $result")
+	} else {
+		log.debug("Couldn't zwave.parse '$description'")
+	}
+	return result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+	[name: "switch", value: cmd.value ? "on" : "off"]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicSet cmd) {
+	[name: "switch", value: cmd.value ? "on" : "off"]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd) {
+	[name: "switch", value: cmd.value ? "on" : "off"]
+}
+
+def zwaveEvent(physicalgraph.zwave.Command cmd) {
+	log.debug("Commands: ${cmd}")
+	[:]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd, ep = null) {
+	log.info "SensorMultilevelReport: ${cmd}, endpoint: ${ep}"
+	def result = []
+
+	def map = [:]
+	switch (cmd.sensorType) {
+		case 1:
+			map.name = "temperature"
+			def cmdScale = cmd.scale == 1 ? "F" : "C"
+			map.value = convertTemperatureIfNeeded(cmd.scaledSensorValue, cmdScale, cmd.precision)
+			map.unit = getTemperatureScale()
+			break
+		default:
+			map.descriptionText = cmd.toString()
+	}
+	log.debug "SensorMultilevelReport, ${map}, ${map.name}, ${map.value}, ${map.unit}"
+	handleChildEvent(map)
+	result << createEvent(map)
+}
+
+def on() {
+	log.debug "on"
+	delayBetween([
+			zwave.basicV1.basicSet(value: 0xFF).format(),
+			zwave.basicV1.basicGet().format()
+	])
+}
+
+def off() {
+	log.debug "off"
+	delayBetween([
+			zwave.basicV1.basicSet(value: 0x00).format(),
+			zwave.basicV1.basicGet().format()
+	])
+}
+
+def ping() {
+	refresh()
+}
+
+def refresh() {
+	refreshChild()
+	def commands = []
+	commands << zwave.basicV1.basicGet().format()
+	delayBetween(commands)
+}
+
+def createChildDevice(childDthNamespace, childDthName, childDni, childComponentLabel) {
```

Comment:
you only ever call this one time with one set of arguments, you should probably just hard-code them inside the function